src/promptflow/tests/sdk_cli_azure_test/e2etests/test_telemetry.py

```diff
@@ -405,3 +405,54 @@ def assert_flow_test(*args, **kwargs):
                 mock_logger.side_effect = assert_flow_test
 
                 pf.flows.test(temp_dir, inputs={"key": "API_BASE"})
+
+    @pytest.mark.skip(reason="Live mode can run successfully, but an error will be reported when recording.")
```

Comment:
Do we have a task to track the recording issue?

Reply:
Yes, It has already been opened to Zhengfei.